doc-content/drools-docs/src/main/asciidoc/GettingStartedDecServices/creating-users-proc.adoc

```diff
@@ -24,7 +24,7 @@ In {CENTRAL}, you can use groups and roles to control permissions for a collecti
 . Click the *Groups* tab and click *Add to groups*.
 . Select `approver` and `kie-server`, and click *Add to selected groups*.
 . Click *Save*.
-. Select `Bill` from the user list, click *Edit* -> *Roles* -> *Add roles*.
+. Select `Bill` from the user list and click *Edit* -> *Roles* -> *Add roles*.
```

Comment:
In the doc preview **_Click the Users tab,_** is present at the beginning of this step. Which one is correct?
```suggestion
. Click the *User* tab, select `Bill` from the user list and click *Edit* -> *Roles* -> *Add roles*.
```
![image](https://user-images.githubusercontent.com/43639538/132723124-6e7af09f-c6cc-4c71-b05b-e97926ff6644.png)